src/platforms/common/performance/index.mdx

```diff
@@ -27,6 +27,16 @@ If you're adding tracing, enable and configure it as documented here. If you’r
 
 With [performance monitoring](/product/performance/), Sentry tracks your software performance, measuring metrics like throughput and latency, and displaying the impact of errors across multiple systems. Sentry captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services. Learn more about our model in [Distributed Tracing](/product/performance/distributed-tracing/).
 
+<PlatformSection supported={["java"]} notSupported={["android", "java.spring", "java.spring-boot"]}>
```

Comment:
this is already on the page - like so. I'd suggest we don't need to duplicate:
![Screen Shot 2021-02-08 at 12 03 53 PM](https://user-images.githubusercontent.com/61481573/107274765-b80a5e00-6a05-11eb-9a6c-4022628e8c6f.png)

Reply:
This is on @bruno-garcia request to highlight that we have dedicated performance feature for Spring and Spring Boot. The part from the screenshot is only about linking errors with spans. I am fine with removing this part as I believe our users know which framework they use, but I'll leave it up to @bruno-garcia

Reply:
How about we modify the existing note (I'll suggest the edit in the comments for those lines), and remove this one?

Reply:
We could delete this since at the top of this page there's already the same note.
I feel like it's a nitpick but `<Note` renders gray text which seems like:

> Don't read this, it isn't important.

But that's unrelated to this change. Lets remove this one and merge